utils/utils.jsx

```diff
@@ -67,6 +67,9 @@ export function cmdOrCtrlPressed(e, allowAlt = false) {
 }
 
 export function isKeyPressed(event, key) {
+    if (event.keyCode === 229) {
```

Comment:
could we move this magic number to a constant that names it?

Reply:
Yes you are right.